parameters.cpp

```diff
@@ -296,6 +317,7 @@ bool Parameters::addParameters(){
    Readparameters::add("AMR.box_center_x","x coordinate of the center of the box that is refined (for testing)",0.0);
    Readparameters::add("AMR.box_center_y","y coordinate of the center of the box that is refined (for testing)",0.0);
    Readparameters::add("AMR.box_center_z","z coordinate of the center of the box that is refined (for testing)",0.0);
+   Readparameters::addComposing("AMR.filterpasses", std::string()+"AMR filter passes for each individual refinement level");
```

Comment:
Why the `std::string()+` here @kstppd?

Reply:
I would suggest this should better be ```std::string("text")```.

I understand what's happening here: if you *just* put a ```const char *``` there, in the form of a C string literal, that gets typeconverted into an integer type at compile time, so you end up with a integer-type parameter. Making sure that the datatype is ```std::string``` avoids that. But constructing a ```std::string``` by using the ```+``` operator is a bit odd. :)

Reply:
Oh, I see that that's been done for other calls to `addComposing`, I never noticed that (or when it changed).

Reply:
But not for the IO parameters at the top of this function. So I don't see why this is needed here.